packages/filesystem/src/browser/file-tree/file-tree-model.ts

```diff
@@ -200,10 +201,10 @@ export class FileTreeModel extends CompressedTreeModel implements LocationServic
 
     protected async shouldReplace(fileName: string): Promise<boolean> {
         const dialog = new ConfirmDialog({
-            title: 'Replace file',
-            msg: `File '${fileName}' already exists in the destination folder. Do you want to replace it?`,
-            ok: 'Yes',
-            cancel: 'No'
+            title: nls.localize('theia/filesystem/replaceTitle', 'Replace file'),
+            msg: nls.localize('theia/filesystem/fileVariableInput', 'File \'{0}\' already exists in the destination folder. Do you want to replace it?', fileName),
+            ok: nls.localizeByDefault('Yes'),
+            cancel: nls.localizeByDefault('No')
```

Comment:
I believe we can use the exported constants which will already be localized:

```suggestion
            ok: Dialog.Yes,
            cancel: Dialog.No
```